js/stateless.js/src/rpc.ts

```diff
@@ -148,15 +108,16 @@ export class Rpc extends Connection implements CompressionApiInterface {
   async getUtxos(
     owner: PublicKey,
     config?: GetUtxoConfig,
-  ): Promise<WithMerkleUpdateContext<UtxoWithMerkleContext>[]> {
+  ): Promise<WithMerkleUpdateContext<CompressedAccountWithMerkleContext>[]> {
     const unsafeRes = await rpcRequest(this.rpcEndpoint, 'getUtxos', [
       owner.toString(),
       config?.encoding || 'base64',
     ]);
-    const res = create(
-      unsafeRes,
-      jsonRpcResultAndContext(nullable(UtxosResult)),
-    );
+    // const res: any = create(
+    //     unsafeRes,
+    //     jsonRpcResultAndContext(nullable(UtxosResult)),
+    // );
```

Comment:
leftover comment?

Reply:
will be needed, i will sync rpc-interface with photon in a different pr